### ISSUE TITLE
CMD+Q doesn't quit app

### ISSUE THREAD

**Issue by ocean-mars** [zen-browser/desktop#1824]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

didn't even customize any keyboard shortcut

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.6

### What platform are you seeing the problem on?

macOS - Intel

**Comment by sander-p** [comment]
Actually, none of the keyboard shortcuts under the Zen Browser menu are working. I'm on MacOS (Intel) as well.

**Comment by AryanDesh** [comment]
Other than the predefined shortcut , the user defined ones dont work. I'm on windows

**Comment by aleksanderbl29** [comment]
> Actually, none of the keyboard shortcuts under the Zen Browser menu are working. I'm on MacOS (Intel) as well.

Same here. Only toggling the sidebar width and opening/closing tabs works on my M1 mac.

**Comment by Zegnat** [comment]
Looks like a recent version broke all shortcuts. Whether they are set by Zen Browser or just Firefox defaults. I came looking for this issue when I noticed I could no longer minimize windows with ⌘M (CMD+M). In the Keyboard Shortcuts section of the settings it was set to Ctrl+M, which would have been wrong to start with, but that did not work for me either. (Changing it to ⌘M had no effect.)

Instead ⌘M seems to mute the current tab? I have no idea where that got defined as the shortcut.

When I went digging, it also turned out ⌘, (comma) does not work to take me to the Settings.

**Comment by dploeger** [comment]
Same on ARM btw

**Comment by Zegnat** [comment]
Looks like I was too quick to comment. Apparently I was not on 1.0.1-a.7 yet. I just updated and now ⌘M works, and is correctly listed under Keyboard Shortcuts. With that said, most of everything else is still broken (⌘H to hide; ⌘, to open settings; etc.).

M2, macOS 15.0 (24A335), Zen Browser 1.0.1-a.7 (64-bit)

**Comment by 00y300** [comment]
I can also add that command Q or command comma does not work with the latest build of zen.

**Comment by piavgh** [comment]
Same here, I'm not sure why a fork of Firefox does not have that functionality while Firefox does ...

**Comment by kalvinpearce** [comment]
There is a potential workaround that I commented in another issue https://github.com/zen-browser/desktop/issues/1669#issuecomment-2393731282
tldr; unset the affected shortcuts in the Zen Settings > Keyboard Shortcuts

**Comment by piavgh** [comment]
> There is a potential workaround that I commented in another issue [#1669 (comment)](https://github.com/zen-browser/desktop/issues/1669#issuecomment-2393731282) tldr; unset the affected shortcuts in the Zen Settings > Keyboard Shortcuts

This works. Thanks a lot

**Comment by mauro-balades** [comment]
Ill move this to https://github.com/zen-browser/desktop/issues/1629, thanks
